src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -160,6 +166,18 @@ private void readFromNBT(NBTTagCompound compound)
             }
         }
 
+        // Restore colony achievements
+        NBTTagList achievementTagList = compound.getTagList(TAG_ACHIEVEMENT_LIST, NBT.TAG_COMPOUND);
+        this.colonyAchievements = new ArrayList<>();
+        for (int i = 0; i < achievementTagList.tagCount(); ++i)
+        {
+            NBTTagCompound achievementCompound = achievementTagList.getCompoundTagAt(i);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'achievementCompound' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)